.travis.yml

```diff
@@ -24,17 +24,17 @@ before_install:
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then 2to3 -nw pysal/ > /dev/null; fi
 
 install:
-  - conda install --yes numpy scipy nose pip
+  - conda install --yes pip
   - if [ "$PYSAL_PLUS" == true ]; then
-        echo 'plus testing'; conda install --yes --file requirements_plus.txt;
-        else conda install --yes --file requirements.txt; 
+        echo 'plus testing'; pip install .[plus];
+    else pip install .;
     fi;
-  - pip install -r travis.txt
+  - pip install .[dev]
 
```

Comment:
This change, from using conda to using pip, might be what's causing travis to fail, since: 

```
$> source create -n gdaltest numpy
$> source activate gdaltest
$> conda install --yes fiona
```
completes successfully.